src/main/java/ru/yandex/qatools/ashot/comparison/ImageDiffer.java

```diff
@@ -79,6 +85,10 @@ protected void markDiffPoints(Screenshot expected, Screenshot actual, ImageDiff
     }
 
     private boolean hasDiffInChannel(Screenshot expected, Screenshot actual, int i, int j) {
+        if(ignoredColor != null && rgbCompare(expected.getImage().getRGB(i, j), ignoredColor.getRGB(), 0)) {
```

Comment:
As I see, if we will not define ignoredColor, `hasDiffInChannel` will always be true?